node/forest_libp2p/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [dependencies]
 utils = { path = "../utils" }
 networks = { path = "../../types/networks" }
+fil_types = { path = "../../types" }
```

Comment:
sorry for the nit, but can you avoid bringing this dependency into the libp2p crate? Since the type is as simple as just `git_version!()`, maybe just have that in place of `CURRENT_COMMIT`

Reply:
> Hey @nannick, this PR looks great and I'd like to pull it in. The branch just needs to be updated and that last change ([#977 (comment)](https://github.com/ChainSafe/forest/pull/977#discussion_r569568138)) made. If you give me privileges to modify your branch I can do this for you, if you'd like :)

Sorry for the late fix on such a small PR